### PR TITLE
[hotfix][Flink] Address Flink DIM and hive documentation formats

### PR DESCRIPTION
**Pull request by YesOrNo828**
## Why are the changes needed?

Address the Flink Dim and hive documentation format.

## Brief change log

  - *change the documentation of flink-dim.md and flink-quick-start.md*

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (yes / no) no 
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented) docs

### CODE REVIEW

File: site/docs/ch/flink/flink-dim.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Flink SQL 官方支持多种 Join 方式，见 [官方文档](https://nightlies.
 对于 Flink 官方 Regular Join、Interval Join、Temporal Join，用户可以按照上述 Flink 官方的文档正常使用。
 对于将 Arctic 表作为维表的 Lookup Join 场景，请按本文档的描述配置使用。
 
-**未来会基于 Arctic log-store 实现更实时的维表**
+**未来会基于 Arctic LogStore 实现更实时的维表**
```
Comment (zhoujinsong) [inline site/docs/ch/flink/flink-dim.md:8]: How a bout use  word `Logstore` ?

File: site/docs/ch/flink/flink-get-started.md [hunk 2]
```diff
@@ -74,6 +74,7 @@ cp ../flink-shaded-hadoop-2-uber-${HADOOP_VERSION}-10.0.jar lib
 
 ## Hive兼容
 Arctic 0.3.1 版本开始支持 Hive 兼容的功能，可以通过 Flink 读取/写入 Arctic Hive 兼容表数据。当通过 Flink 操作 Hive 兼容表时，需要注意以下几点：
+
 1. Flink Runtime Jar 不包括 Hive 依赖的 Jar 包内容，需要手动将[ Hive 依赖的 Jar 包](https://repo1.maven.org/maven2/org/apache/hive/hive-exec/2.1.1/hive-exec-2.1.1.jar)放到 flink/lib 目录下；
```
Comment (zhoujinsong) [inline site/docs/ch/flink/flink-get-started.md:78]: Arctic DO NOT dependent on `hive-exec`, why should Flink need this dependency?

**Review by zhoujinsong — COMMENTED** [review]
@YesOrNo828  I left some comments. PTAT

File: site/docs/ch/flink/flink-get-started.md (hunk 2 above)
Comment (YesOrNo828) [reply site/docs/ch/flink/flink-get-started.md:78]: The flink-runtime module has excluded the arctic-hive/hive-metastore dependency and shaded the parquet and avro dependency. So flink-runtime is running with hive-exec succeed.

**Review by zhoujinsong — APPROVED** [review]
LGTM
